custom/eqa/expressions.py

```diff
@@ -79,6 +79,44 @@ def __call__(self, item, context=None):
         }
 
 
+class EQAActionItemSpec(JsonObject):
+    type = TypeProperty('cqi_action_item')
+    xmlns = StringProperty()
+    section = StringProperty()
+
+    def __call__(self, item, context=None):
+        xforms_ids = CaseAccessors(item['domain']).get_case_xform_ids(item['_id'])
+        forms = FormAccessors(item['domain']).get_forms(xforms_ids)
+        f_forms = [f for f in forms if f.xmlns == self.xmlns]
+        s_forms = sorted(f_forms, key=lambda x: x.received_on)
+
+        if len(s_forms) > 0:
+            latest_form = s_forms[-1]
+        else:
+            latest_form = None
+        path = 'action_plan/%s/action_plan/%s'
+        incorrect_question = latest_form.get_date(path % (self.section, 'incorrect_questions'))
+        if latest_form:
+            support = ', '.join(
+                [
+                    item.get_case_property(x.strip()) for x in
+                    latest_form.get_date(path % (self.section, 'copy-1-of-responsible')).split(',')
+                ]
+            )
+            return {
+                'gap': latest_form.get_data('code_to_text/%s' % incorrect_question),
```

Comment:
We actually want a slightly different value here. `code_to_text/<incorrect_question>` is a label node, and we want the label of that label from the `corehq.apps.app_manager.xform.XForm` model. As you currently have it here, I believe `gap` will always be the XFormInstance value of the label node, which will always be "OK", or something like that. You can get the value like this:
```python
question_list = Application.get(latest_form.app_id).get_questions()
questions = {x['value']: x for x in question_list}
gap = questions['code_to_text/%s' % incorrect_question].label
```
(this snippet of code is untested)